hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/distributions/dstable/DistributionSetTable.java

```diff
@@ -208,17 +209,11 @@ public void drop(final DragAndDropEvent event) {
         };
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
I would move the `@SuppressWarnings("unchecked")` to line `215` because as I can see this is only line which is unchecked and not the whole method.